src/src/com/task30/Main.java

```diff
@@ -0,0 +1,29 @@
+package src.com.task30;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+
+public class Main {
+    public static void main(String[] args) {
+        int a = 10;
+        ArrayList<Integer> randomNumbers = new ArrayList<>(a);
+
+        for (int i = 0; i < a; i++) {
+            int random = (int) (Math.random() * 10);
+            randomNumbers.add(random);
+        }
+        System.out.println(randomNumbers);
+
+        for (int i = 0; i < randomNumbers.size(); i++) {
```

Comment:
Removing could be made with Set collection, Or we have a lot of methods in Collections util class